Shooting Star Expansion (#5619)
endless-sky/endless-sky: data/persons.txt

	frequency 20000
	government "Author"
	personality

> 		forbearing mining surveillance unconstrained swarming

Reviewer: Surveillance and swarming don't really mix

Author: Given that it already has mining, I'll remove the tactical and asteroid scan power as well as the surveillance personality

	system
		attributes "ember waste"
	phrase
		word
			"Long years of technological development were for me a pathway to many abilities some consider to be unnatural."
			"What were you thinking about when you hailed me? 'Oh look, an asteroid. Let's chat.'"
			"How does it feel to try hailing an asteroid and actually succeeding?"
			"What a day to be a non-sentient rock floating in the middle of the eternal void."
			"When even the local asteroid belt starts chatting with you, you can tell you've been jumping a little too much lately."
			"What's weirder, an asteroid that emits purple rays or the person that tried to talk with it?"
			"My goals are beyond your understanding."
			"What the Quarg say about the Builders is false. I am the only one responsible for their extinction, and I am proud of it."
			"You think that nuclear weapons are bad? Come closer and you will see what a real weapon is."

> 			"I have been classified as a mass destruction weapon by most human and alien governments. You don't want to see what I am capable of."

Reviewer: The phrasing in English is generally "weapon of mass destruction." Is it intentional to be "mass destruction weapon"? If so, we may need a file-level comment to prevent it from being changed (and even then we may still get reports about it).

Author: Changed to weapon of mass destruction

> 			"Sometimes, just for fun, I set myself on a close trajectory on an inhabited planet and pass just above their atmosphere. I love how lesser species start to freak out as they think that the end is near."

Reviewer: ```suggestion
			"Sometimes, just for fun, I set myself on a close trajectory with an inhabited planet, passing just above their atmosphere. I love how lesser species freak out when they think their end is near."
```

> 			"I have been watching you, <first> <last> from New Boston, current captain of the <ship>."

Reviewer: Alas, "from New Boston" will not always be accurate.

Author: Removed, at least for now

			"I am here for you."
			"I want your money."
			"Three Skeleton Key. Three Skeleton Key. Three Skeleton Key. Three Skeleton Key. Three Skeleton Key. Three Skeleton Key. Three Skeleton Key. Three Skeleton Key."
			"Your politics bore me."
			"There is only one being in the whole universe that isn't commanded by AI, and that's me."

> 			"Shields are useless, I can pass through them just fine. I can show you if you want."

Reviewer: ```suggestion
			"Shields are useless--I can pass through them just fine. I can show you, if you'd like."
```

			"The Quarg are wrong saying that the firstborns are long gone, for I am the firstborn."
			"Have you ever been in the Weft? It's so damn boring. It's a lot more fun to mess with simpler minds like yours."

> 			"I remember when the Archons tried to kill me blowing up some supernovae. They got so depressed when they understood that not even their weapons could stop me."

Reviewer: ```suggestion
			"I remember when the Archons tried to kill me with some supernovae. They got so depressed when they understood that not even their weapons could stop me."
```

> 			"I ate a black hole once. It took me a few years, but it was definetly worth it. Tastes like chicken."

Reviewer: ```suggestion
			"I ate a black hole once. It took me a few years, but it was definitely worth it. Tastes like chicken."
```

			"I am the only thing in the universe capable of beating the power of friendship."

> 			"I want to tell you some words of a long gone past, which back then used to scare people of your race more than any weapon or war machine. These words are, 'The MacOS workflow run was not successful'."

Reviewer 2: ```suggestion
			"I want to tell you some words of a long gone past, which back then used to scare people of your race more than any weapon or war machine. These words are, 'The MacOS workflow run was not successful.'"
```

		word
			">"
	ship "Shooting Star" "Prototype B3-CC4"

ship "Shooting Star"

> 	plural "Shootings Star"

Reviewer 2: ?

	noun "genocidal maniac"
	sprite "ship/shooting star/shooting star"
		"frame time" 3
	attributes
		category "Heavy Warship"

> 		"hull" 31500

Reviewer 2: Maybe make this have a non-standard disabled threshold just to mess around. This would drop its threshold from 15% to 5%.
```suggestion
		"hull" 31500
		"threshold percentage" 0.05
```

> 		"automaton" 200

Author: Given that the command center removes one from "automaton", I thought that doing this I would have prevented it from ever going under 1 (or even close), thus making it impossible to be used as flagship

> 		"automaton" 200

Reviewer: That's fair, and should have the intended effect.

> 		"cost" -49500000

Reviewer 2: I could just disown it with no cost to me. I'd just ditch the negative cost.

Author: Eh, I'd rather just keep it, I bet there will be someone who didn't think of that. Me, for instance

		"jump fuel" 50
		"hyperdrive" 1
		"energy generation" 1
		"tactical scan power" 20
		"asteroid scan power" 30
		"atmosphere scan" 100
		"spinal mount" 1
		"crystal projector" 1
		"shooting star" 2
		"core crystal" -2
		weapon

> 			"blast radius" 180000

Reviewer 2: ```suggestion
			"blast radius" 20000
```

			"hull damage" 79999
			"piercing" 1
			"hit force" -8200
	outfits
		"Crystal Projector"
		"Crystalline Formation"
		"Hexagonal Crystal"
		"Floating Crystal"
		"Tree Skeleton Key Stone"
	gun 0 0
	turret 0 0
	turret 0 0
	turret 0 0
	explode "pug tiny explosion" 5
	explode "pug small explosion" 20
	explode "pug medium explosion" 25
	explode "pug large explosion" 15
	"final explode" "final explosion medium"
	description "This ship appears to be a hollowed-out asteroid filled with strange equipment. It is unclear what is creating the rays of light, but they must have something to do with how it moves without obvious engines."
	description "	Its advanced properties also allow it to reject the influence of enemy command centers, nullifying their effect."

outfit "Crystal Projector"
	category "Systems"
	"cost" 1492
	thumbnail "outfit/rock 0"
	"mass" 2
	"outfit space" -2
	"gun ports" -1
	"crystal projector" -1
	weapon
		"hardpoint sprite" "hardpoint/shooting star flare/ss-rays"
			"frame time" 3

> 	description "Never underestimate the power of a bunch of crystals on a rock. In this case all they do is projecting some purple harmless rays around the Shooting Star, but the point still stands."

Reviewer: ```suggestion
	description "Never underestimate the power of a bunch of crystals on a rock. In this case all they do is project some harmless purple rays around the Shooting Star, but the point still stands."
```

outfit "Floating Crystal"
	category "Systems"
	"cost" 1861
	thumbnail "outfit/rock 3"
	"mass" 3
	"outfit space" -3
	"turret mounts" -1
	"spinal mount" -1
	weapon
		"turret turn" 6
		"velocity" 250
		"lifetime" 3
		"reload" 4
		"firing force" -93
		"hit force" -81

> 	description "The real core of the Shooting Star, this floating crystal has the power of using the effects of space-time to reduce the distance between a ship and its target. Legend has it this is no different from a thruster, but conventional engines don't bring your enemy closer."

Reviewer: ```suggestion
	description "The real core of the Shooting Star, this floating crystal uses the effects of space-time to reduce the distance between a ship and its target. Legend has it this is no different from a thruster, but conventional engines don't bring your enemy closer."
```

		"reload" 1
		"piercing" 1
		"hull damage" 293

> 	description "There appears to have been various attempts in the past to discover the workings of this outfit, but the debate is still ongoing wether this can be considered a device or not."

Reviewer: ```suggestion
	description "There appear to have been several prior attempts to discern the mechanics of this outfit, but the debate is still ongoing whether this can be considered a device or not."
```

		"firing force" 2700
		"hit force" 3500
		"hull damage" 2160
	description "'Any sufficiently advanced technology is indistinguishable from magic,' Arthur C. Clarke said. This is sufficiently advanced technology."

> outfit "Tree Skeleton Key Stone"

Reviewer: 😂
